### PR TITLE
issue templates

### PR DESCRIPTION
**Pull request by luckedea**
## Description

Basic issue templates, including:
- config
- bug report
- feature requests
- compiling/ building
- and extensions (unfinished)

Related discussion https://github.com/nzbgetcom/nzbget/discussions/157

## Testing

[Cloned repo](https://github.com/luckedea/nzbgetissuestest/issues/new/choose)

### CODE REVIEW

**Comment by dnzbk** [comment]
Looks good to me.

File: .github/ISSUE_TEMPLATE/02_feature_request.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  attributes:
+    label: Is there already an issue for this request?
+    description: Please make sure you are not creating an already submitted <a href="https://github.com/nzbgetcom/nzbget/issues">Issue</a>. Check closed issues as well, because your issue could have been resolved and waiting to be released.
+    options:
+      - label: I have checked older issues, open and closed
+        required: true
+- type: dropdown
+  attributes:
+    label: Platform
+    description: Select a specific platform for this request, choose Any if it applies to all platforms
```
Comment (phnzb) [inline .github/ISSUE_TEMPLATE/02_feature_request.yml:15]: Any -> All

File: .github/ISSUE_TEMPLATE/01_bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    - v23-testing
+    - v22-stable (nzbgetcom takeover)
+    - v22-testing (nzbgetcom takeover)
+    - v21 or earlier (orignal nzbget)
+  validations:
+    required: true
+- type: dropdown
+  attributes:
+    label: Platform
+    description: Select a specific platform for this bug report, choose Any if it applies to all platforms
```
Comment (phnzb) [inline .github/ISSUE_TEMPLATE/01_bug_report.yml:27]: Any -> All

File: .github/ISSUE_TEMPLATE/01_bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    - macOS
+    - NAS/Synology/QNAP
+    - Linux/Docker
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Environment
+    description: Please provide the details of the system NZBGet is running on. The more information you can provide, the better
+    placeholder: |
```
Comment (phnzb) [inline .github/ISSUE_TEMPLATE/01_bug_report.yml:40]: Maybe separate sections ?
- Platform (windows/linux/nas)
- Architecture (arm / aarch64 / x86 / x86_64 etc)
- OS version
- Docker: yes/no

File: .github/ISSUE_TEMPLATE/03_build.yml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    - v23-testing
+    - v22-stable (nzbgetcom takeover)
+    - v22-testing (nzbgetcom takeover)
+    - v21 or earlier (orignal nzbget)
+  validations:
+    required: true
+- type: dropdown
+  attributes:
+    label: Platform
+    description: Select a specific platform for this issue, choose Any if it applies to all platforms
```
Comment (phnzb) [inline .github/ISSUE_TEMPLATE/03_build.yml:20]: Any -> All

File: .github/ISSUE_TEMPLATE/03_build.yml [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    - macOS
+    - NAS/Synology/QNAP
+    - Linux/Docker
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Environment
+    description: Please provide the details of the system where the issue is observed
+    placeholder: |
```
Comment (phnzb) [inline .github/ISSUE_TEMPLATE/03_build.yml:33]: Separate sections ?

Comment (luckedea) [reply .github/ISSUE_TEMPLATE/03_build.yml:33]: Sure, but later.

File: .github/ISSUE_TEMPLATE/01_bug_report.yml (hunk 3 above)
Comment (luckedea) [reply .github/ISSUE_TEMPLATE/01_bug_report.yml:40]: Good suggestion, but let's do it later to keep it simple.

**Review by phnzb — APPROVED** [review]

**Pull request merged**